src/codegen.cpp

```diff
@@ -2243,7 +2243,7 @@ static jl_array_t* build_stack_crumbs(jl_codectx_t &ctx) JL_NOTSAFEPOINT
             break;
         }
         if (caller) {
-            assert(ctx.emission_context.enqueuers.count(caller) == 1);
+            // assert(ctx.emission_context.enqueuers.count(caller) == 1);
```

Comment:
Can you add a comment that explains why this is commented out and under what circumstances it could/should be uncommented? If it's not something that's important to reenable in the future, perhaps delete it rather than comment it out?